documentation/src/main/resources/pages/ditto/httpapi-concepts.md

```diff
@@ -138,6 +139,29 @@ This maps to the following HTTP API endpoints:
 * `/policies/{policyId}/entries/{entryLabel-1}/resources`: accessing the resources of a single `Policy` entry with the
   label `{entryLabel-1}`
 
+#### `/checkPermissions` in API 2
+The checkPermissions endpoint verifies permissions for specified entities on various resources, ensuring they have the necessary access rights as defined in the policies.
```

Comment:
I think we should move this to the new `check-permissions.html` page.  
Especially the example.

And then maybe only link to that page inside this section.